--- libical-3.0.6/cmake/modules/GtkDoc.cmake.orig	2019-09-14 18:43:47.000000000 +0200
+++ libical-3.0.6/cmake/modules/GtkDoc.cmake	2019-10-15 05:35:40.930446559 +0200
@@ -49,7 +49,7 @@
     ${CMAKE_CURRENT_BINARY_DIR}/${_module}-docs.sgml
     @ONLY)
 
-  set(OUTPUT_DOCDIR ${SHARE_INSTALL_DIR}/gtk-doc/html/${_module})
+  set(OUTPUT_DOCDIR ${SHARE_INSTALL_DIR}/doc/gtk-doc/html/${_module})
 
   set(_filedeps)
   set(_srcdirs)
