--- libical-3.0.16/src/python/CMakeLists.txt.orig	2023-03-04 19:38:26.490595543 +0100
+++ libical-3.0.16/src/python/CMakeLists.txt	2023-03-04 19:41:03.049747390 +0100
@@ -6,8 +6,19 @@ include_directories(
 
 ########### next target ###############
 
+find_package(PythonInterp REQUIRED)
+find_package(PythonLibs REQUIRED)
+include_directories(${PYTHON_INCLUDE_DIRS} ${CMAKE_BINARY_DIR}/src ${CMAKE_BINARY_DIR})
+add_custom_command(
+  OUTPUT
+  ${CMAKE_BINARY_DIR}/src/python/LibicalWrap.c
+  COMMAND
+  swig -python -Wall -I${CMAKE_SOURCE_DIR}/src -I${CMAKE_BINARY_DIR}/src -o ${CMAKE_BINARY_DIR}/src/python/LibicalWrap.c ${CMAKE_SOURCE_DIR}/src/python/LibicalWrap.i
+  DEPENDS ${CMAKE_SOURCE_DIR}/src/python/LibicalWrap.i
+)
+
 set(LibicalWrap_LIB_SRCS
-  LibicalWrap.c
+  ${CMAKE_BINARY_DIR}/src/python/LibicalWrap.c
 )
 
 add_library(LibicalWrap ${LIBRARY_TYPE} ${LibicalWrap_LIB_SRCS})
@@ -15,9 +26,8 @@ add_library(LibicalWrap ${LIBRARY_TYPE}
 target_link_libraries(LibicalWrap)
 
 set_target_properties(LibicalWrap PROPERTIES
-  VERSION ${LIBICAL_LIB_VERSION_STRING}
-  SOVERSION ${LIBICAL_LIB_MAJOR_VERSION}
-)
-install(TARGETS LibicalWrap DESTINATION lib)
+  OUTPUT_NAME "_LibicalWrap" PREFIX "")
+set(PY_SITEDIR lib${LIB_SUFFIX}/python CACHE STRING "Python module dir")
+install(TARGETS LibicalWrap DESTINATION ${PY_SITEDIR})
 
 ########### install files ###############
--- libical-3.0.6/src/CMakeLists.txt.orig	2019-10-15 05:33:01.947974507 +0200
+++ libical-3.0.6/src/CMakeLists.txt	2019-10-15 05:33:59.810994370 +0200
@@ -4,6 +4,7 @@
 if(ICAL_GLIB)
   add_subdirectory(libical-glib)
 endif()
+add_subdirectory(python)
 if(LIBICAL_BUILD_TESTING)
   add_subdirectory(test)
 endif()
